ui/js/directives/treeherder/clonejobs.js

```diff
@@ -898,60 +902,46 @@ treeherder.directive('thCloneJobs', [
         var generateJobElements = function(resultsetAggregateId, resultset) {
 
             var tableEl = $('#' + resultsetAggregateId);
-
             var waitSpanEl = $(tableEl).prev();
             $(waitSpanEl).css('display', 'none');
-
-            var name, option, platformId, platformKey, row, platformTd, jobTdEl, j;
-            for (j=0; j<resultset.platforms.length; j++) {
-
+            var name, option, platformId, platformKey, j;
```

Comment:
I believe `j` is no longer necessary. The other variables should be assigned when they're used, not here.